Task1.PianoKeyboard/Enums/EOctave.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Task1.PianoKeyboard.Enums
+{
+    internal enum EOctave
```

Comment:
Do not use "E" prefix, it is not common practice, https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces